federation/pkg/federation-controller/configmap/configmap_controller_test.go

```diff
@@ -107,6 +112,7 @@ func TestConfigMapController(t *testing.T) {
 	// Test update federated configmap.
 	configmap1.Annotations = map[string]string{
 		"A": "B",
+		"federation.beta.kubernetes.io/cluster-selector": "{\"location\": \"europe\"}",
```

Comment:
I would argue against using beta until the criteria for federation being considered beta is both determined and met.